nvme-library/Cargo.lock

```diff
@@ -0,0 +1,349 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file should not be checked in, we use a single cargo.lock for the whole of mayastor.

Reply:
OK. I'll undo that